src/test/functional/pages/BuzzPage.ts

```diff
@@ -1,32 +1,21 @@
 import { BasePage } from "./BasePage";
-import { Locator, Page } from "@playwright/test";
 export class BuzzPage extends BasePage {
-    readonly sharePhotosButton: Locator;
-    readonly fileInput: Locator;
-    readonly submitPhoto: Locator
-    readonly threeDotsIcon: Locator
-    readonly deletePostParagraph: Locator
-    readonly confirmDeleteButton: Locator
-    readonly photoImg: Locator
-    readonly noPostParagraph: Locator
-    readonly buzzPageButton: Locator
-  
-    constructor(page: Page) {
-      super(page)
-      this.sharePhotosButton = page.locator('button:has-text("Share Photos")');
-      this.fileInput = page.locator('input[type="file"]');
-      this.submitPhoto = page.locator('button.oxd-button--main:has-text("Share")');
-      this.threeDotsIcon = page.locator('i.oxd-icon.bi-three-dots');
-      this.deletePostParagraph = page.locator('p:has-text("Delete Post")');
-      this.confirmDeleteButton = page.locator('button:has-text("Yes, Delete")');
-      this.photoImg = page.locator('.orangehrm-buzz-photos-item img')
-      this.noPostParagraph = page.locator('p:has-text("No Posts Available")')
-      this.buzzPageButton = page.locator('span:has-text("Buzz")')
-    }
-
-    async sharePhotos(): Promise<void> {
-    const filePath = '../test/functional/files/file1.png';
-        await this.sharePhotosButton.click();
+    public readonly sharePhotoButton = this.page.getByRole("button", { name: "Share Photos" })
```

Comment:
When I added a comment suggesting using roles I didn't mean only this one... please review your locators one by one and try to use locators recommended by the playwright. As I said - if you have a problem we can sync and I can help, but I won't paste every single one of them here for you to just use them because you'll learn nothing from it.

Reply:
BTW. I see you have changed a name to singular - if button says "Share Photos", you should use it as a name...
I also see that you have marked comment about submitPhoto as resolved - I just unresolved it.

Reply:
it is impossible to use getByRole in those:
    public readonly deletePostParagraph = this.page.getByText("Delete Post"); 
    public readonly editPostParagraph = this.page.getByText("Edit Post");